src/globals.h

```diff
@@ -40,7 +40,27 @@
 #define BATTERY_MONITOR BAT_INTERNAL
 #endif
 
-#if defined(LED_INVERTED) && LED_INVERTED
+// If LED_PIN is not defined in "defines.h" take the default pin from "pins_arduino.h" framework.
+// If there is no pin defined for the board, use LED_PIN 255 and disable LED
+#if !defined(LED_PIN)
+#if defined(LED_BUILTIN)
+#define LED_PIN LED_BUILTIN
+#define ENABLE_LEDS true
```

Comment:
```suggestion
        #define ENABLE_LEDS true
```

Also, what if `ENABLE_LEDS` is set to `false` in `defines.h`?